addons/account/models/ir_actions_report.py

```diff
@@ -21,7 +21,12 @@ def _post_pdf(self, save_in_attachment, pdf_content=None, res_ids=None):
             res_ids = None
             if not save_in_attachment:
                 raise UserError(_("No original vendor bills could be found for any of the selected vendor bills."))
-        return super(IrActionsReport, self)._post_pdf(save_in_attachment, pdf_content=pdf_content, res_ids=res_ids)
+        res = super()._post_pdf(save_in_attachment, pdf_content, res_ids)
+        if self.model == 'account.move':
+            moves = self.env[self.model].browse(res_ids)
+            for move in moves:
```

Comment:
As the issue only occurs for Customer Invoices, maybe you can filter here:
```suggestion
            for move in moves.filtered(lambda m: m.type == 'out_invoice'):
```